packages/adaptive_scaffold/example/lib/main.dart

```diff
@@ -445,42 +606,44 @@ class _ItemListTile extends StatelessWidget {
         // than large screens.
         // Small screens open a modal with the detail view while large screens
         // simply show the details on the secondaryBody.
-        selectCard(_all_Items.indexOf(item));
+        selectCard(allItems.indexOf(item));
         if (!Breakpoints.mediumAndUp.isActive(context)) {
           Navigator.of(context).pushNamed(
             _ExtractRouteArguments.routeName,
             arguments: _ScreenArguments(item: item, selectCard: selectCard),
           );
         } else {
-          selectCard(_all_Items.indexOf(item));
+          selectCard(allItems.indexOf(item));
         }
       },
       child: Padding(
         padding: const EdgeInsets.all(8.0),
         child: Container(
           decoration: BoxDecoration(
-              color: selected == _all_Items.indexOf(item)
-                  ? const Color.fromARGB(255, 234, 222, 255)
-                  : const Color.fromARGB(255, 243, 237, 247),
+              color: selected == allItems.indexOf(item)
+                  ? const Color.fromARGB(255, 237, 221, 255)
```

Comment:
You may want to keep the colors from the previous version so the design is as close to the M3 example as possible.